experimenter/experimenter/experiments/models.py

```diff
@@ -209,6 +209,9 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
     prevent_pref_conflicts = models.BooleanField(blank=True, null=True, default=False)
     proposed_release_date = models.DateField(blank=True, null=True)
 
+    is_localized = models.BooleanField(default=False)
+    localizations = models.TextField(blank=True, null=True)
```

Comment:
we can use JSONField here, thoughts 🤔 ?

Reply:
Exactly correct @brennie but a good thought @yashikakhurana 👍   We used to use JSON field for this but we had to move away from it.